gef-ui/server/utils/helpers.test.js

```diff
@@ -696,6 +721,72 @@ describe('mapSummaryList()', () => {
       expect(href).toContain('/unissued-facilities/');
       expect(href).toContain('/change');
     });
+
+    it('isUsingFacilityEndDate', () => {
+      const mockedDisplayItemsEnd = [
+        {
+          label: 'Is using facility end date',
+          id: 'isUsingFacilityEndDate',
+        },
+      ];
+
+      const facility = { ...MOCK_ISSUED_FACILITY, details: { ...MOCK_ISSUED_FACILITY.details, isUsingFacilityEndDate: true } };
+
+      const mapSummaryParams = {
+        app: MOCK_AIN_APPLICATION,
+        user: MOCK_REQUEST,
+      };
+
+      const { text, href } = mapSummaryList(facility, mockedDisplayItemsEnd, mapSummaryParams, true)[0].actions.items[0];
+      // should be allowed to change so should display change
+      expect(text).toEqual('Change');
+      expect(href).toContain('/unissued-facilities/');
+      expect(href).toContain('/change');
+    });
+
+    it('bankReviewDate', () => {
+      const mockedDisplayItemsEnd = [
+        {
+          label: 'Bank review date',
+          id: 'bankReviewDate',
+        },
+      ];
+
+      const facility = { ...MOCK_ISSUED_FACILITY, details: { ...MOCK_ISSUED_FACILITY.details, bankReviewDate: new Date().toISOString() } };
+
+      const mapSummaryParams = {
+        app: MOCK_AIN_APPLICATION,
+        user: MOCK_REQUEST,
+      };
+
+      const { text, href } = mapSummaryList(facility, mockedDisplayItemsEnd, mapSummaryParams, true)[0].actions.items[0];
+      // should be allowed to change so should display change
+      expect(text).toEqual('Change');
+      expect(href).toContain('/unissued-facilities/');
+      expect(href).toContain('/change');
+    });
+
+    it('facilityEndDate', () => {
+      const mockedDisplayItemsEnd = [
+        {
+          label: 'Facility end date',
+          id: 'facilityEndDate',
+        },
+      ];
+
+      const facility = { ...MOCK_ISSUED_FACILITY, details: { ...MOCK_ISSUED_FACILITY.details, facilityEndDate: new Date().toISOString() } };
+
+      const mapSummaryParams = {
+        app: MOCK_AIN_APPLICATION,
+        user: MOCK_REQUEST,
+      };
+
+      const { text, href } = mapSummaryList(facility, mockedDisplayItemsEnd, mapSummaryParams, true)[0].actions.items[0];
+      // should be allowed to change so should display change
+      expect(text).toEqual('Change');
+      expect(href).toContain('/unissued-facilities/');
+      expect(href).toContain('/change');
+    });
   });
 
   /**
```

Comment:
It looks the describe name `maps and returns summary list with change button for relevant rows for facilities changed to issued on preview page` features twice in this file on line 664 and on 797 - is there an accidental set of duplicated tests? Or does one need renaming?

Reply:
Looks like one was issued facilities, the other unissued. Have updated the describe